index.html

```diff
@@ -556,6 +581,29 @@
 The [=self-dealing=] [=purposes=] are often justified as bankrolling the pro-social outcomes
 but this requires collective oversight to be [=appropriate=].
 
+### Group Privacy {#group-privacy}
+
+There are different ways for [=people=] to become members of a group. Either they can join it
+deliberately, making it a self-constituted group such as when joining a club, or they can be
+classified into it by an external party, typically a bureaucracy or its computerised equivalent
+([[?Beyond-Individual]]). In the latter case, [=people=] may not be aware that they are being
+grouped together, and the definition of the group may not be intelligible (for instance if it is
+created from opaque machine learning techniques).
+
+Protecting group privacy can take place at two different levels. [=People=] may wish to protect
+knowledge that they are members of the group even though the existence of the group and its actions
+may be well knows (eg. membership in a dissidents movement under authoritarian rule). Conversely,
+the existence of a group or at least its activities may need to be protected even in cases in which
+its members are guaranteed to remain anonymous. An example [=privacy violation=] for the latter case
```

Comment:
I'd swap these two: making everything secret seems less complicated than making half the things secret.

Reply:
I don't have a preference, so happy to go with yours.